static/css/siglens.css

```diff
@@ -3169,6 +3169,8 @@ float: right;
 }
 
 .search-db-input,.metrics-input{
+    padding-left: 1rem;
+    padding-right: 3.4rem;
     height: 32px;
```

Comment:
can you increase the padding-right to 4.4rem?

Reply:
yes i can.